charts/fluent-operator/values.yaml

```diff
@@ -0,0 +1,182 @@
+  # You can adjust it based on the log volume.
+  resources:
+    limits:
+      cpu: 500m
+      memory: 200Mi
+    requests:
+      cpu: 10m
+      memory: 25Mi
+  # Specify custom annotations to be added to each FluentBit pod.
+  annotations: {}
+    # prometheus.io/scrape: "true"
+  ## Reference to one or more secrets to be used when pulling images
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+  imagePullSecrets: []
+  # - name: "image-pull-secret"
+  secrets: []
+  # List of volumes that can be mounted by containers belonging to the pod.
+  volumes: []
+  # Pod volumes to mount into the container's filesystem.
+  volumesMounts: []
+
+  # Remove the above empty volumes and volumesMounts, and then set volumes and volumesMounts as below if you want to collect node exporter metrics
+# volumes:
+#   - name: hostProc
+#     hostPath:
+#       path: /proc/
+#   - name: hostSys
+#     hostPath:
+#       path: /sys/
+# volumesMounts:
+#   - mountPath: /host/sys
+#     mountPropagation: HostToContainer
+#     name: hostSys
+#     readOnly: true
+#   - mountPath: /host/proc
+#     mountPropagation: HostToContainer
+#     name: hostProc
+#     readOnly: true
+
+  # Set a limit of memory that Tail plugin can use when appending data to the Engine.
+  # If the limit is reach, it will be paused; when the data is flushed it resumes.
+  # if the inbound traffic is less than 2.4Mbps, setting memBufLimit to 5MB is enough
+  # if the inbound traffic is less than 4.0Mbps, setting memBufLimit to 10MB is enough
+  # if the inbound traffic is less than 13.64Mbps, setting memBufLimit to 50MB is enough
+  input:
+    tail:
+      memBufLimit: 5MB
+    # uncomment below nodeExporterMetrics section if you want to collect node exporter metrics
+#   nodeExporterMetrics:
+#     tag: node_metrics
+#     scrapeInterval: 15s
+#     path:
+#       procfs: /host/proc
+#       sysfs: /host/sys
+
+  # Configure the output plugin parameter in FluentBit.
+  # You can set enable to true to output logs to the specified location.
+  output:
+    es:
+      enable: false
+      host: "<Elasticsearch url like elasticsearch-logging-data.kubesphere-logging-system.svc>"
+      port: 9200
+      logstashPrefix: ks-logstash-log
+#      path: ""
+#      bufferSize: "4KB"
+#      index: "fluent-bit"
+#      httpUser:
+#      httpPassword:
+#      logstashFormat: true
+#      replaceDots: false
+#      enableTLS: false
+#      tls:
+#        verify: On
+#        debug: 1
+#        caFile: "<Absolute path to CA certificate file>"
+#        caPath: "<Absolute path to scan for certificate files>"
+#        crtFile: "<Absolute path to private Key file>"
+#        keyFile: "<Absolute path to private Key file>"
+#        keyPassword:
+#        vhost: "<Hostname to be used for TLS SNI extension>"
+    kafka:
+      enable: false
+      brokers: "<kafka broker list like xxx.xxx.xxx.xxx:9092,yyy.yyy.yyy.yyy:9092>"
+      topics: ks-log
+
```

Comment:
I would add the `stdout` plugin here, particularly if we add the exclude annotation above. I think it is super helpful, particularly for new users even if commented out just to show them how to easily add it.

Reply:
Personally I would add options for at least some of the other popular outputs too, even commented out e.g. Opensearch, Loki, Datadog, etc.

Metrics collection and output could be handled in a separate values file potentially or maybe we should add defaults here for at least Fluent Bit internal metrics and a Prometheus endpoint along with a commented out Prometheus Remote Write.

Reply:
`# You can find more supported output plugins here: https://github.com/fluent/fluent-operator/tree/master/docs/plugins/fluentbit/clusteroutput`

Reply:
> Metrics collection and output could be handled in a separate values file potentially or maybe we should add defaults here for at least Fluent Bit internal metrics and a Prometheus endpoint along with a commented out Prometheus Remote Write.

We can do this in another PR, adding Fluent Bit internal metrics is ok. But adding a Prometheus endpoint to a daemonset is not reasonable because different pods of the daemonset will try to scape the same metrics and we can do this in the new added `collector` statefulset: https://github.com/fluent/fluent-operator/pull/484#issuecomment-1338667600

Reply:
Agreed on the extra values file.

Unrelated but that linked PR has no real description or details other than we're deploying it as a statefulset (which always makes me nervous anyway). Be good to explain what the PR is doing rather than have to read the comments and code.

I still would like to see a stdout option, too many people try to debug via Kibana/Grafana/etc. when the issue may be nothing to do with FB.

Reply:
> Unrelated but that linked PR has no real description or details other than we're deploying it as a statefulset (which always makes me nervous anyway). Be good to explain what the PR is doing rather than have to read the comments and code.

we'll add docs the the FluentBit collector mode later

Reply:
> I still would like to see a stdout option, too many people try to debug via Kibana/Grafana/etc. when the issue may be nothing to do with FB.

We can do this by adding the commented settings to exclude FB logs and the setting of output to stdout @wenchajun